docs/containers-storage.conf.5.md

```diff
@@ -48,6 +48,11 @@ The `storage` table supports the following options:
   Maximum size of a container image.  Default is 10GB.  This flag can be used to set quota
   on the size of container images.
 
+**override_kernel_check**=""
+  Tell storage drivers to ignore kernel version checks.  Some storage drivers assume that if a kernel is too
+  old,that the driver is not supported.  But for kernels that have had the drivers backported, this flag
```

Comment:
Needs whitespace in old,that.  Aren't these part of the [storage.options] table?

Reply:
Typo while we're in here: congtainer.

Reply:
Fixed